src/test/components/ActiveTabTimeline.test.js

```diff
@@ -93,7 +93,7 @@ describe('ActiveTabTimeline', function() {
       if (track.type !== 'tab') {
         throw new Error('Expected a tab track.');
       }
-      const threadIndex = track.mainThreadIndex;
+      const threadIndex = (track.threadIndexes.values().next().value: any);
```

Comment:
Suggestion: This `any` coercion is pretty risky, and may introduce an run-time error. This would be better to wrap up in a type-safe function in `src/utils/flow.js` and then properly handle the `void` case.

```
export function getFirstItemFromSet<T>(set: Set<T>): T | void {
  return set.values().next().value;
}
```

Reply:
Update: Oh, I just realized this is in a test, so the risk is much lower, but it would be good not to coerce to any. So the above suggestion on adding a function is optional. I know we have this awkward `value().next().value` pattern a few times in the codebase, so it might be worth fixing with a good utility function.